ci/test.yml

```diff
@@ -38,6 +38,26 @@ core unix unit test:
     - cd core
     - pipenv run make test
 
+core unix device ui test:
+  stage: test
+  <<: *only_changes_core
+  dependencies:
+    - core unix frozen regular build
+  script:
+    - cd core
+    - pipenv run make test_emu_ui
+    - cp /var/tmp/trezor.log ${CI_PROJECT_DIR}
+    - cd ../ci
+    - pipenv run python prepare_ui_artifacts.py
+  artifacts:
+    name: core-unix-device-ui-test
+    paths:
+      - trezor.log
+      - ci/tmp/
```

Comment:
maybe a more distinct directory name for this?

Reply:
Initial idea was we could use this for smth else as well. But it probably does not make sense. So `ui_test_fixtures` or `ui_test_records` ?

Reply:
honestly whichever. `ui_test_records` seems a little nicer

Reply:
Fixed in 362dbce60